docs/training-apis/api-pytorch-advanced.txt

```diff
@@ -0,0 +1,147 @@
+########################
+ PyTorch API - Advanced
+########################
+
+*******************
+ Gradient Clipping
+*******************
+
+Users need to pass a gradient clipping function to
+:meth:`~determined.pytorch.PyTorchTrialContext.step_optimizer`.
+
+.. _pytorch-custom-reducers:
+
+******************
+ Reducing Metrics
+******************
+
+Determined supports proper reduction of arbitrary training and validation metrics, even during
+distributed training, by allowing users to define custom reducers. Custom reducers can be either a
+function or an implementation of the :class:`determined.pytorch.MetricReducer` interface. See
+:meth:`determined.pytorch.PyTorchTrialContext.wrap_reducer` for more details.
```

Comment:
This section was written as an introduction to the rest of the metric reducers API.  Why did you separate the introduction from the contents of the API?  This says basically nothing by itself.

Reply:
This is not separated from the contents of the API. This is to add a brief topic guide for linking reducing metrics to the reference guide of which function or class you should use. In the future, this will add more content about what the concept of metric reducing is and when and why you want to do this. Currently, this is to just be separate from the basic concepts which new users would use to implement the basic functionality of PyTorch API. Send me comments on how you think it should be written and we will add more content in the future.